### PR TITLE
Make p2p bindings available in conductor API

### PR DESCRIPTION
**Pull request by ddd-mtl**
Added p2p_bindings() getter in conductor API

### CODE REVIEW

File: cli/src/cli/run.rs [hunk 1]
```diff
@@ -32,6 +32,11 @@ pub fn run(
         .load_config()
         .map_err(|err| format_err!("{}", err))?;
 
+    println!(
```
Comment (Connoropolous) [inline cli/src/cli/run.rs]: Won't this be a weird message if p2p_bindings is None?

Comment (ddd-mtl) [reply cli/src/cli/run.rs]: Yeah, kind of. Us lazy devs are used to weird messages like that :)

Comment (Connoropolous) [reply cli/src/cli/run.rs]: Could you make it conditional?

Comment (Connoropolous) [reply cli/src/cli/run.rs]: Us obsessive documenters go nuts over messages like that :p

Comment (Connoropolous) [reply cli/src/cli/run.rs]: You can use [ci skip] and [skip travis] in the commit message to prevent rebuilding

Comment (ddd-mtl) [reply cli/src/cli/run.rs]: i'll just erase it

**Pull request merged**